src/reanimated2/screenTransition/animationManager.ts

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+import type { ScreenTransitionConfig } from './commonTypes';
+import { configureProps } from '../../ConfigHelper';
+import { applyStyle } from './styleUpdater';
+import { swipeSimulator } from './swipeSimulator';
+
+configureProps();
```

Comment:
Why do we need this call here?

Reply:
Normally, the `configureProps` method is called in `createAnimatedComponent()` when someone uses an animated component like `Animated.View`. However, in the case of screen animation, there is no need to use any animated component. Without calling `configureProps`, we cannot perform style updates because all props will be treated as "JSProps".